ansible/configs/sap-integration/default_vars.yml

```diff
@@ -0,0 +1,384 @@
+
+# The output_dir holds all of the files generated during the deployment
+# This includes generated Heat templates, SSH config, SSH keys
+# This must be an absolute path and no vars (like $HOME or ~)
+output_dir: /tmp/output_dir
+
+# The {{ guid }} is used everywhere and it is what differentiates otherwise
+# identical environments. Make this unique. Usually they are 4 characters, but
+# it can be any reasonable length.
+guid: notset
+
+# The type of cloud provider this will be deployed to
+cloud_provider: osp
+
+# Authenication credentials for OpenStack in order to create the things.
+# These should be included with your secrets, but are listed here for reference
+# osp_auth_url:
+# osp_auth_username:
+# osp_auth_password:
+# osp_auth_cloud:
+# osp_auth_project_domain: #usually set to "default"
+# osp_auth_user_domain: #usually set to "default"
+
+# This is an account that must exist in OpenStack.
+# It is used to create projects, access, Heat templates
+admin_user: opentlc-mgr
+
+# This is the user that Ansible will use to connect to the nodes it is
+# configuring from the admin/control host
+ansible_user: cloud-user
+
+# This needs to be True for 4.3 and earlier.
+# For OpenShift 4.4 and later it can be false
+osp_use_swift: false
```

Comment:
i don't think you are doing anything with OpenShift. If so, get rid of this var.